the-list-be/.prettierrc.json

```
@@ -0,0 +1,11 @@
+{
+  "printWidth": 80,

```

Предлагаю избегать скобок в arrow-функциях (в конфиг для фронта надо будет тоже это добавить) (-:

Если примем этот изменение, надо будет преттиер перезапустить накоде.

```suggestion
  "printWidth": 80,
  "arrowParens": "avoid",
```

@Move-Fast-and-Break-Things-Study-Group/the-list-core, что вы думаете?

`(ctx) => {`
vs
`ctx => {`

😂

---

без скобок выглядит понятнее 
я видел, что в каком-то примере в скобках было несколько функций, как это работает?